components/dashboard/src/teams/Members.tsx

```diff
@@ -18,40 +22,131 @@ export default function() {
     const location = useLocation();
     const team = getCurrentTeam(location, teams);
     const [ members, setMembers ] = useState<TeamMemberInfo[]>([]);
+    const [ genericInvite, setGenericInvite ] = useState<TeamMembershipInvite>();
+    const [ showInviteModal, setShowInviteModal ] = useState<boolean>(false);
 
     useEffect(() => {
         if (!team) {
             return;
         }
         (async () => {
-            const infos = await getGitpodService().server.getTeamMembers(team.id);
+            let [infos, invite] = await Promise.all([
+                getGitpodService().server.getTeamMembers(team.id),
+                getGitpodService().server.getGenericInvite(team.id)]);
+
             setMembers(infos);
+            if (!invite) {
+                invite = await getGitpodService().server.resetGenericInvite(team.id);
```

Comment:
Why calling `reset` is nothing was found?

Reply:
If there is no invite URL we want to create one.